mockgcp/common/operations/http.go

```diff
@@ -0,0 +1,61 @@
+package operations
+
+import (
+	"context"
+	"net/http"
+
+	"cloud.google.com/go/longrunning/autogen/longrunningpb"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/common/httpmux"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"k8s.io/klog/v2"
+)
+
+func (s *Operations) RegisterOperationsHandler(prefix string) func(ctx context.Context, mux *runtime.ServeMux, conn *grpc.ClientConn) error {
+	return func(ctx context.Context, mux *runtime.ServeMux, conn *grpc.ClientConn) error {
+		forwardResponseOptions := mux.GetForwardResponseOptions()
+
+		// GET /{prefix}/operations/{name}
+		if err := mux.HandlePath("GET", "/"+prefix+"/operations/{name}", func(w http.ResponseWriter, r *http.Request, pathParams map[string]string) {
```

Comment:
Any reason for string concatenation over sprintf?

Reply:
My personal style is to use concatenation when it is just concatenation of a reasonable number of strings (go doesn't have string auto-conversion, so it has to be strings).  When there's a int -> string involved, then I find %d much more readable than strconv.Itoa etc